### DISCUSSION TITLE
Switch some ACU Apps back to PKGBUILDS from AUR?

### DISCUSSION THREAD

**Discussion by fraz0815** [kwankiu/acu#15] [General]
Hi,
thanks for the great Arch Image.
I am wondering whats the deal using acu and (partly outdated & broken) yaml repos instead of PKGBUILD's, e.g.:

Communication: Signal (no offcial ARM64 release as of July 2024). Compiling with acu ends up with a lot of different problems like old signal version, dependency hell ...
vs AUR: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=signal-desktop-beta (builds an runs fine on ARM64).

Thanks, Fraz

**Comment by kwankiu** [comment]
> Hi, thanks for the great Arch Image. I am wondering whats the deal using acu and (partly outdated & broken) yaml repos instead of PKGBUILD's, e.g.:
> 
> Communication: Signal (no offcial ARM64 release as of July 2024). Compiling with acu ends up with a lot of different problems like old signal version, dependency hell ... vs AUR: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=signal-desktop-beta (builds an runs fine on ARM64).
> 
> Thanks, Fraz

It is installing `signal-desktop-arm` from AUR which seems to be removed/unmaintained now, thanks for pointing out `signal-desktop-beta`, I will update that soon.

**Reply by kwankiu** [reply]
@fraz0815, the dependency hell seems to be an issue of `fpm`, both `signal-desktop-arm` and `signal-desktop-beta` requires `fpm`, I just tried building it and neither `acu`, `yay` nor `paru` can resolve the dependency. 

How did you builds and runs fine with `signal-desktop-beta` ?
